docs/source/developer/caching.rst

```diff
@@ -13,17 +13,17 @@ The Implementation
 
 Caching is done by saving the compiled *object code*, the ELF object of the
 executable code.  By using the *object code*, cached functions have minimal
-overhead because no compilation is needed. The cached data is saved under
+overhead because no compilation is needed. The cached data is saved under the
 ``__pycache__`` directory. The index of the cache is stored in a ``.nbi``
-file.  One index per function. It lists all the overloaded signatures compiled
-for the function. The *object code* is stored in files of ``.nbc`` extension.
-One per overload.
+file, with one index per function, and it lists all the overloaded signatures compiled
```

Comment:
this line is too long now, sorry, that was the auto-suggest thing not line wrapping